include/sway/tree/node.h

```diff
@@ -17,6 +17,8 @@ enum sway_node_type {
 	N_CONTAINER,
 };
 
+#define NUM_NODE_TYPES ((size_t)(N_CONTAINER + 1))
```

Comment:
We use N_LAST instead of this

Reply:
That is, we add an enum member, not a macro

Reply:
(Not a fan of adding an enum member TBH)

Reply:
Let's discuss that out of band later, we can't be inconsistent just for this one case